pages/index.tsx

```diff
@@ -31,7 +31,7 @@ const Home: NextPage = () => {
     if (maybeCached) {
       analysis = JSON.parse(maybeCached);
     } else {
-      const res = await fetch(`/api/scanner/${pkgName}`);
+      const res = await fetch(`/api/scanner/${pkgName.replace("/", "%2F")}`);
```

Comment:
I thought about creating a helper function as we use the same twice. But it seems already really expressive. WDYT?

Reply:
We could make it afterward anyway ^^